### ISSUE TITLE
EntryChange is not serializable

### ISSUE THREAD

**Issue by zaynetro** [javers/javers#468]
`MapChange` contains a list of entry changes (`EntryChange`) which is not serializable. This breaks `Diff` object serialization when there are some map changes.

I can prepare a pull request if you confirm that this issue needs to be fixed.

Although the only change needed is this:

```
--- a/javers-core/src/main/java/org/javers/core/diff/changetype/map/EntryChange.java
+++ b/javers-core/src/main/java/org/javers/core/diff/changetype/map/EntryChange.java
@@ -2,6 +2,7 @@ package org.javers.core.diff.changetype.map;
 
 import org.javers.core.diff.changetype.Atomic;
 
+import java.io.Serializable;
 import java.util.Objects;
 
 import static org.javers.common.validation.Validate.argumentIsNotNull;
@@ -9,7 +10,7 @@ import static org.javers.common.validation.Validate.argumentIsNotNull;
 /**
  * @author bartosz walacik
  */
-public abstract class EntryChange {
+public abstract class EntryChange implements Serializable {
     private final Atomic key;
 
     EntryChange(Object key) {
```

**Comment by bartoszwalacik** [comment]
Ok, create a Pull Request, we will merge

**Comment by bartoszwalacik** [comment]
released in 2.7.2
